cli.js

```diff
@@ -17,4 +17,14 @@ var cli = meow({
 	string: ['_']
 });
 
-cpy([cli.input[0]], cli.input[1], {overwrite: cli.flags.overwrite});
+try {
+	cpy([cli.input[0]], cli.input[1], {overwrite: cli.flags.overwrite}, function (err) {
+		if (err) {
+			console.error(err.toLocaleString());
```

Comment:
It has, because errors are a objects: [Object.prototype.toLocaleString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/toLocaleString),
but your suggestion is much cleaner.